cartography/data/indexes.cypher

```diff
@@ -112,7 +115,8 @@ CREATE INDEX ON :RedshiftCluster(arn);
 CREATE INDEX ON :RDSInstance(db_instance_identifier);
 CREATE INDEX ON :RDSInstance(id);
 CREATE INDEX ON :RDSInstance(arn);
-CREATE INDEX ON :ReplyUri(id)
+CREATE INDEX ON :ReplyUri(id);
+CREATE INDEX ON :RestAPI(id);
```

Comment:
Let's rename it to APIGatewayRestAPI for clarity

Reply:
done